lua/xwarn/commands/sv_warn.lua

```diff
@@ -28,7 +28,7 @@ xAdmin.Core.RegisterCommand("warn", "Warn a user", 30, function(admin, args)
 	end
 
 	xWarn.Database.CreateWarn(target, (IsValid(targetPly) and targetPly:Name()) or "Unknown", admin:SteamID64(), admin:Name(), reason, nil, function(_, q)
-		xAdmin.Core.Msg({admin, " warned ", ((IsValid(targetPly) and targetPly) or target), " for: ", Color(255, 0, 0), reason, color_white, " (", Color(128, 0, 128), q:lastInsert(), color_white, ")"})
+		xAdmin.Core.Msg({admin, " warned ", ((IsValid(targetPly) and targetPly) or target), " for: ", xWarn.Config.WarnReasonColor, reason})
```

Comment:
Ideally we want to keep the warning ID in the actual message, it makes it easy to delete an accidental warn without having to go through someone's warns.

Could just make it a config option as some servers might not need it.